splink/internals/charts.py

```diff
@@ -84,7 +84,7 @@ def save_offline_chart(
         print(iframe_message.format(filename=filename))  # noqa: T201
 
 
-def match_weights_chart(records, as_dict=False):
+def partial_match_weights_chart(records, as_dict=False):
```

Comment:
I think I'd leave this as `match_weights_chart` just for compatibility with the public API - having two names that do the same thing is a bit confusing (though I agree that this name is more precise)